manager/state/raft/wait.go

```diff
@@ -50,15 +50,17 @@ func (w *wait) trigger(id uint64, x interface{}) bool {
 	return false
 }
 
-func (w *wait) cancel(id uint64) bool {
+func (w *wait) cancel(id uint64) {
 	w.l.Lock()
 	waitItem, ok := w.m[id]
 	delete(w.m, id)
 	w.l.Unlock()
 	if ok && waitItem.cancel != nil {
 		waitItem.cancel()
 	}
-	return ok
+	if waitItem.ch != nil {
```

Comment:
I'm still confused how is this possible. But it fails with `nil channel close` on TestStress.